crypto/rsa/rsa_pmeth.c

```diff
@@ -864,7 +877,10 @@ static int pkey_pss_init(EVP_PKEY_CTX *ctx)
         return 0;
 
     /* See if minimum salt length exceeds maximum possible */
-    max_saltlen = RSA_size(rsa) - EVP_MD_get_size(md);
+    md_size = EVP_MD_get_size(md);
+    if (md_size <= 0)
```

Comment:
I think raising RSA_R_INVALID_DIGEST_LENGTH is appropriate here too.

Reply:
Thanks. I have submitted a new commit to add all the needed ERR_raise().